js-core/homeworks/homework-10(3)/src/main.js

```diff
@@ -0,0 +1,59 @@
+// var inp = document.createElement('input');
+// var XMLS = new XMLSerializer();
+// var inp_xmls = XMLS.serializeToString(inp); // Конвертируем DOM эелемент в сторку
+
+
+// // Вставляет вновь созданный элемент в тело документа
+// let elem = document.body.insertAdjacentHTML('afterbegin', inp_xmls);
+// console.log(elem);
+var ezjQuery = {
+    temp: [],
+    temp3: document.getElementById("list"),
+    add: function(tag, text) {
+        text = text || '';
+        let elem = document.createElement(tag);
```

Comment:
you don't have to create new element tag... probably you just need to create a string :)...